napari/_vispy/tests/test_vispy_camera.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+from napari import Viewer
+from vispy.scene import PanZoomCamera, ArcballCamera
+
+
+def test_camera(qtbot):
+    """Test vispy camera model interaction."""
+    viewer = Viewer()
+    view = viewer.window.qt_viewer
```

Comment:
I don't think this is a big deal, but if you wanted to get change all of the `view.view.*` to `view.*`, you could probably do 
```suggestion
    view = viewer.window.qt_viewer.view
```

Reply:
did some renaming to try and improve